model/src/main/kotlin/utils/SimplePackageConfigurationProvider.kt

```diff
@@ -43,7 +46,15 @@ class SimplePackageConfigurationProvider(
         fun forDirectory(directory: File): SimplePackageConfigurationProvider {
             val entries = directory.walkBottomUp()
                 .filterTo(mutableListOf()) { !it.isHidden && it.isFile }
-                .map { it.readValue<PackageConfiguration>() }
+                .map { file ->
+                    try {
+                        file.readValue<PackageConfiguration>()
+                    } catch (e: IOException) {
+                        log.error { "Error reading package configuration from '${file.absoluteFile}'.'" }
```

Comment:
Superfluous single quote at the end in `'"`.

Reply:
Done.